agent/ansible/pbench/agent/galaxy.yml

```diff
@@ -0,0 +1,57 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: pbench
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: agent
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.2
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- The Pbench team
+
+
+### OPTIONAL but strongly recommended
+
+# A short summary description of the collection
+description: Roles that help install pbench-agent
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- GPL-3.0-or-later
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: ''
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags: [pbench]
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+# range specifiers can be set and are separated by ','
+dependencies: {}
+
+# The URL of the originating SCM repository
+repository: http://github.com/distributed-system-analysis/pbench
+
+# The URL to any online docs
+documentation: http://github.com/distributed-system-analysis/pbench
+
+# The URL to the homepage of the collection/project
+homepage: http://github.com/distributed-system-analysis/pbench
```

Comment:
Since this is for the agent, perhaps we should reference the `agent` subtree of the repository?